sound/hybrid_font.h

```diff
@@ -234,11 +234,38 @@ class HybridFont : public SaberBase, public Looper {
     EnableAmplifier();
     if (!f->files_found()) return RefPtr<BufferedWavPlayer>(nullptr);
     RefPtr<BufferedWavPlayer> player = GetFreeWavPlayer();
```

Comment:
We should probably break out the stuff below into a new function, something like:

RefPtr<BufferedWavPlayer> GetOrFreeWavPlayer(Effect* e);

Then we can replace this call with a call to the new function.

Reply:
Attempt made, although I'm not 100% that the filtering out only the wavplayers playing the effect is foolproof. 
I did have a 1 in 1000 occasion where humm was stopped to free up a player, and that's unacceptable of course.

Reply:
done